### PR TITLE
Add Redsys

### PR DESCRIPTION
**Pull request by davidlj95**
image

RedSys is a popular, widely used payment gateway in Spain

**Issue:** none

**Popularity metric:**

- [3.524 global rank, top 85 in Spain in SimilarWeb](https://www.similarweb.com/website/redsys.es/)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

## Brand guidelines
Couldn't find any brand related information in their main website:
https://redsys.es/

## SVG
Found a vectorial image from it's gradient-like logo in the mentioned source URL
https://norbr.com/payment-providers/redsys/

Which is extracted from their main logo:

image

This official one can be found at https://redsys.es/img/logo.png

Then used Inkscape to isolate the shape, simplify path, layers, remove fill color, ...

## Color
Their logo uses an orange gradient as can seen above. In order to choose one orange colour, looked at their site. Found 2 main colours used in their site:

### FFA500
As navigation bar bottom border

image

As search icon color

image

As highlight color

image

Colours extracted from:
- https://pagosonline.redsys.es/index.html

### DC7C26
Used in some sections:

image

As link colour:

image

As footer icon colour:

image

Colours extracted from:
- https://redsys.es/en/rsc.html

Eventually went with `DC7C26` as appears in main site, not `pagosonline.redsys.es` and seems a bit closer to the original logo's gradient colour scheme.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -12733,6 +12733,11 @@
             "hex": "000000",
             "source": "https://github.com/redox-os/assets/blob/4935a777cd7aa44323d3c263b1e0bb4ae864a027/logos/redox/vectorized_logo.svg"
         },
+        {
+            "title": "Redsys",
+            "hex": "DC7C26",
+            "source": "https://norbr.com/payment-providers/redsys/"
```
Comment (adamrusted) [inline _data/simple-icons.json:12739]:
```suggestion
            "source": "https://redsys.es"
```

File: icons/redsys.svg
Comment (adamrusted) [inline icons/redsys.svg:1]: I'm happy enough with this treatment, see below comparison with the PNG in the website header.
Given it's a manual vectorisation I'll need another approval though. I'd also suggest the change of source to an official source. Thanks for the work on this @davidlj95!

Comment (davidlj95) [reply icons/redsys.svg:1]: Thanks!!

Comment (adamrusted) [reply icons/redsys.svg:1]: Given we've had no responses - I'm going to look at merging this in - thanks again for your work @davidlj95!

**Pull request merged**